extensions/wikia/EditAccount/SpecialEditAccount_body.php

```diff
@@ -411,6 +414,10 @@ public static function closeAccount( $user = '', $changeReason = '', &$mStatusMs
 		// if they are connected from facebook, disconnect them
 		self::disconnectFromFacebook( $user );
 
+		/** @var HeliosClient $heliosClient */
+		$heliosClient = Injector::getInjector()->get(HeliosClient::class);
+		$heliosClient->forceLogout($user->getId());
```

Comment:
I would think about the case when request fail. What will happen? Whole process will be reverted? Partially done? Will there be a sane message to the user or 5xx?